src/parser/transform/transform_standalone_call.cpp

```diff
@@ -1,14 +1,20 @@
 #include "parser/standalone_call.h"
+#include "parser/standalone_call_function.h"
 #include "parser/transformer.h"
 
 namespace kuzu {
 namespace parser {
 
 std::unique_ptr<Statement> Transformer::transformStandaloneCall(
     CypherParser::KU_StandaloneCallContext& ctx) {
-    auto optionName = transformSymbolicName(*ctx.oC_SymbolicName());
-    auto parameter = transformExpression(*ctx.oC_Expression());
-    return std::make_unique<StandaloneCall>(std::move(optionName), std::move(parameter));
+    if (ctx.oC_FunctionInvocation()) {
+        return std::make_unique<StandaloneCallFunction>(
+            transformFunctionInvocation(*ctx.oC_FunctionInvocation()));
+    } else {
+        auto optionName = transformSymbolicName(*ctx.oC_SymbolicName());
+        auto parameter = transformExpression(*ctx.oC_Expression());
+        return std::make_unique<StandaloneCall>(std::move(optionName), std::move(parameter));
```

Comment:
This is fine but we might want to differentiate more between `StandaloneCall & StandaloneCallFunction`. Maybe call them `StandaloneCallFunction & StandaloneCallSetting`